client/src/App.css

```diff
@@ -1,6 +1,12 @@
-body {
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+
+* {
   padding: 0;
   margin: 0;
```

Comment:
padding and margin are not inherited (ie. is 0 for every element by default, regardless of the parent's component), can be removed from here

Reply:
It's for resetting the default margins and paddings which the browser adds to the elements

Reply:
default margin and padding is zero